ui/pages/bridge/quotes/bridge-quotes-modal.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
+import { IconName } from '@metamask/snaps-sdk/jsx';
+import {
+  Box,
+  ButtonIcon,
+  ButtonIconSize,
+  Modal,
+  ModalContent,
+  ModalHeader,
+  ModalOverlay,
+  Text,
+} from '../../../components/component-library';
+import {
+  TextAlign,
+  TextVariant,
+} from '../../../helpers/constants/design-system';
+import { getBridgeQuotes } from '../../../ducks/bridge/selectors';
+import { getQuoteDisplayData } from '../utils/quote';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+
+export const BridgeQuotesModal = ({
+  onClose,
+  ...modalProps
+}: Omit<React.ComponentProps<typeof Modal>, 'children'>) => {
+  const { quotes } = useSelector(getBridgeQuotes);
+  const t = useI18nContext();
+
+  const [, setSortOrder] = useState(t('bridgeOverallCost'));
```

Comment:
What's the point of setting a sort order if the state is never consumed?

Reply:
I added this bc eslint doesn't like the empty `onClick` handler. Good callout tho, I'll remove the onClick prop completely since it's not required and we can't sort until the code for fetching dest token exchange rates is available anyway (doing all this separately to keep this change small)